### PR TITLE
HCAP-1375 - Readme proofreading and formatting

### PR DESCRIPTION
**Pull request by rhattie-ey**
[HCAP-1375](https://eydscanada.atlassian.net/browse/HCAP-1375)

This PR improves the README file with the following changes:

* Fixes typos and formatting issues
* Restructures certain information into tables
* Updates and expands upon ToC

### CODE REVIEW

**Review by fw-noel — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -121,75 +141,64 @@ Go to `http://keycloak.local.freshworks.club:8080/auth` console and add users/as
...
+/employer-confirmation | Redirect here after form submission and display the expression of interest submitted
+/login                 | Portal entry point for authenticated users to upload, view, and manage data
 
 ### Public API Routes
 
-- /employer-form [POST] submit new employer expression of interest form
-- /keycloak-realm-client-info [GET] load config for login
-- /version [GET] view deployed version of the application
-- In production: / [GET] serves the built client app
+Path                        | Method | Description
```
Comment (vanics) [inline README.md:169]: Great improvement!

File: README.md [hunk 2]
```diff
@@ -121,75 +141,64 @@ Go to `http://keycloak.local.freshworks.club:8080/auth` console and add users/as
...
+Route | Exposes a service to the Internet. Routes differ from services in that they may only transmit HTTP(S) traffic. As such, the database service could not be directly exposed to the Internet.
+Deployment Config | Defines how a new version of an application is to be deployed. Additionally, triggers for redeployment are defined within this object. For the HCAP application, we've used a rolling deployment triggered by new images pushed to the image stream and tagged with the `latest` tag.
+Secret |Defines values that can be used by pods within in the same namespace. While there are no secrets defined in our server application, there is a reference to a secret defined by the [MongoDB database template](openshift/mongo.yml). In order for the server to access the DB, it must be provided with `MONGODB_DATABASE` and `MONGODB_URI` environment variables. The definition for these environment variables can be found in the [server deployment config template](openshift/server.dc.yml). Note that they are referencing the `${APP_NAME}-mongodb` (resolves to `hcap-mongodb`) secret and the `mongo-url` and `database` keys within this secret.
```
Comment (vanics) [inline README.md:194]: Nice!

**Review by vanics — APPROVED** [review]
It looks great! Thank you for your diligence on this.

**Pull request merged**